app/components/OutcomeBar.tsx

```diff
@@ -95,32 +99,63 @@ export const OutcomeBar = ({ market }: OutcomeBarProps) => {
     market.outcomeTokenMarginalPrices?.[1] ?? lastTradeMarginalPrices?.[1]
   );
 
-  const hasOutcomePercentages = outcome0.percentage && outcome1.percentage;
+  const getOutcomesPercentages = () => {
+    if (isAnswerFinal && winnerOutcome) {
+      return winnerOutcome.index ? ['0', '100'] : ['100', '0'];
+    }
+
+    return [outcome0?.percentage, outcome1?.percentage];
+  };
+
+  const outcomesPercentages = getOutcomesPercentages();
+  const outcome0percentage = outcomesPercentages[0];
+  const outcome1percentage = outcomesPercentages[1];
+  const hasOutcomePercentages = outcome0percentage && outcome1percentage;
+  const showOutcome0 = !!Number(outcome0percentage) && isAnswerFinal;
+  const showOutcome1 = !!Number(outcome1percentage) && isAnswerFinal;
```

Comment:
We should call it `isWinnerOutcome0` and maybe check `winnerOutcome.index` instead of `outcome0percentage`.

```suggestion
  const isWinnerOutcome0 = winnerOutcome.index === 0 && isAnswerFinal;
  const isWinnerOutcome1 = winnerOutcome.index === 1 && isAnswerFinal;
```

Reply:
I got it now why it's called show. Maybe we can keep the show, but I think it's cleaner to call the variable what it is instead of what it does.